addons/l10n_ke_edi_tremol/models/account_move.py

```diff
@@ -142,8 +142,13 @@ def is_candidate(discount_line, other_line):
         # The device expects all monetary values in Kenyan Shillings
         if self.currency_id == self.company_id.currency_id:
             currency_rate = 1
+        elif (self.move_type != 'out_invoice'):
+            reversed_lines = self.reversed_entry_id.invoice_line_ids.filtered(
```

Comment:
@dbkosky It is not because it is not an out_invoice that reversed_entry_id is set and it would be good to add a comment. 
```suggestion
        elif self.move_type != 'out_invoice' and self.reversed_entry_id:
            # We suppose the reconciliation will be done with this invoice so in Odoo, so the exchange rate difference
            # account move should compensate for the discrepancy between Odoo and what we send to the gov
            reversed_lines = self.reversed_entry_id.invoice_line_ids.filtered(
```